x-pack/plugin/esql/src/main/antlr/EsqlBaseLexer.g4

```diff
@@ -6,6 +6,7 @@ ENRICH : 'enrich'             -> pushMode(ENRICH_MODE);
 EVAL : 'eval'                 -> pushMode(EXPRESSION_MODE);
 EXPLAIN : 'explain'           -> pushMode(EXPLAIN_MODE);
 FROM : 'from'                 -> pushMode(FROM_MODE);
+METRICS : 'metrics'           -> pushMode(METRICS_MODE);
```

Comment:
Nit: keep the alphabetical order of the commands.

Reply:
++, I pushed https://github.com/elastic/elasticsearch/pull/108115/commits/ba52c4419caf8d803f8b806a9084206b24e98a6b